### PR TITLE
fix #790: [ES] Missing definition

### LINKED ISSUE #790: [ES] Missing definition
- Wiktionary page: https://es.wiktionary.org/wiki/vellosidad

Wikicode:
```

```

Output:
```

```

Expected:
```

```

---

Model link, if any:

### PR DESCRIPTION
**Pull request by lasconic**
- [x] Fixes #790
- [x] Tests added/updated.
- [x] `./check.sh` run and eventual issues fixed.
- [x] Updated CONTRIBUTORS.md, if not already done in the past.

### CODE REVIEW

File: wikidict/render.py [hunk 1]
```diff
@@ -267,6 +267,7 @@ def parse_word(word: str, code: str, locale: str, force: bool = False) -> Word:
     It is disabled by default to speed-up the overall process, but enabled when
     called from get_and_parse_word().
     """
+    code = re.sub(r"(<!--.*?-->)", "", code, flags=re.DOTALL)
     parsed_sections = find_sections(code, locale)
```
Comment (BoboTiG) [inline wikidict/render.py:271]: Shouldn't  we do that in `cleanup()` directly?

Comment (lasconic) [reply wikidict/render.py:271]: Where exactly ?

Comment (BoboTiG) [reply wikidict/render.py:271]: Maybe https://github.com/BoboTiG/ebook-reader-dict/blob/master/wikidict/utils.py#L372?

Comment (lasconic) [reply wikidict/render.py:271]: We call `clean` in `process_templates`, so after we have found the sections. The bug here is that the HTML comment hides the section to WTP I believe, so we don't get the section or the section title and we don't process any templates in it.

**Pull request merged**